File: android/dtos/PosterServiceDTOs/CreatePosterDTO.kt
package net.ipv64.kivop.dtos

//
//  CreatePosterDTO.swift
//  poster-service
//
//  Created by Dennis Sept on 26.11.24.
//
data class CreatePosterDTO (
    var name : String,
    var description : String?,
    var imageBase64 : String, // Bild als Base64-String
)

File: DTOs/Sources/PosterServiceDTOs/CreatePosterDTO.swift
import Foundation

public struct CreatePosterDTO: Codable {
    public var name: String
    public var description: String?
    public var image: Data

    public init(name: String, description: String? = nil, image: Data) {
        self.name = name
        self.description = description
        self.image = image
    }
}

